randovania/gui/ui_files/preset_am2r_patches.ui

```diff
@@ -213,6 +213,27 @@
              <property name="text">
               <string>Ensures all Save Station doors are normal (blue) doors, even with door lock rando enabled.</string>
              </property>
+             <property name="wordWrap">
+              <bool>true</bool>
+             </property>
+            </widget>
+           </item>
+           <item>
+            <widget class="QCheckBox" name="force_blue_labs_check">
+             <property name="text">
+              <string>Unlock Genetics Laboratory Doors</string>
+             </property>
+            </widget>
+           </item>
+           <item>
+            <widget class="QLabel" name="force_blue_labs_check_2">
+             <property name="text">
+              <string>Ensures that the doors in the later parts of Genetics Laboratory are normal (blue) doors, even with door lock rando enabled.
+To be more precise, all rooms after &quot;Hatchling Room Underside&quot; will have their doors changed.</string>
```

Comment:
A small thing here: Do we write region names without quotes but room names with quotes? Looks a litle bit mixed but dunno how other games do it.

Reply:
honestly, no idea. i think each game just kinda does their own.